src/pages/api/search.js

```diff
@@ -155,11 +155,16 @@ export async function searchCasts(query) {
         { ascending: orderAscending }
       )
   } else {
+    const searchTerms = textQuery
```

Comment:
From my testing, it looks like the `media` variable is always undefined even if the user enters a URL. Since we end up going to the `else` case here on L158, I think L42-L138 can be removed.

Reply:
If the `media` variable is always undefined, I think I'd rather fix that in another PR (probs along with a bigger cleanup) than delete it so I'll just leave it for now. It sounds like it would just get skipped anyway, so it shouldn't cause any performance issues. Thanks for paying such close attention!!